include/blogWithComments_functions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$Error = array();
+
+
+function RepopulateComments($info){
+		global $Error;
+	if(isset($_REQUEST['submitComment']) && (sizeof($Error)>0) && isset($_REQUEST[$info])){
+		echo "<input type='text' name=$info value = $_REQUEST[$info] >";
+	}
+	else {
+		echo "<input type='text' name=$info>";
+	}
+}
+
+
+function SubmitComment($Name, $Email, $Comment){
+		$result = dbQuery("
+			INSERT INTO BlogComments(Name, Email, Comment)
+			VALUES('$Name', '$Email', '$Comment')
+		")->fetch();
+}
+
+
+if(isset($_REQUEST['submitComment'])){
```

Comment:
anything in /include should be wrapped inside a function. Because this code is included at the top of every page, you don't want it to actually execute anything, you just want to define functions, and then you call those functions in the main web page (i.e. the page not in the include folder). All code that handles a function submission like this should be outside /include.